src/views/CronTabDetails/CronTabPageTitle.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { DEFAULT_NAMESPACE } from '@crontab-utils/constants';
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { CronTabKind, cronTabModelRef } from '../../models/CronTabModel';
+import { CronTabActions } from '../components/CronTabActions/CronTabActions';
+import { useCronTabTranslation } from '@crontab-utils/hooks/useCronTabTranslation';
+
+type CronTabPageTitleProps = {
+  cronTab: CronTabKind;
+  name: string;
+  namespace: string;
+};
+
+const CronTabPageTitle: React.FC<CronTabPageTitleProps> = ({ cronTab, name, namespace }) => {
+  const { t } = useCronTabTranslation();
+  return (
+    <>
+      <div className="pf-c-page__main-breadcrumb">
+        <Breadcrumb className="pf-c-breadcrumb co-breadcrumb">
+          <BreadcrumbItem>
+            <Link to={`/k8s/ns/${namespace || DEFAULT_NAMESPACE}/${cronTabModelRef}`}>
+              {t('CronTabs')}
+            </Link>
+          </BreadcrumbItem>
+          <BreadcrumbItem>{t('CronTab Details')}</BreadcrumbItem>
+        </Breadcrumb>
+      </div>
+      <div className="co-m-nav-title co-m-nav-title--detail co-m-nav-title--breadcrumbs">
```

Comment:
We should avoid any `co-` prefixed classes since these are internal and not public APIs. They can change and break plugins from version to version. If we need to wait for the SDK to expose details page components, we should probably wait, or just use PF components for now.